diff -urN kopete-4.12.4/protocols/wlm/wlmchatsession.cpp kopete-4.12.4-patched/protocols/wlm/wlmchatsession.cpp
--- kopete-4.12.4/protocols/wlm/wlmchatsession.cpp	2014-03-03 18:20:02.000000000 +1100
+++ kopete-4.12.4-patched/protocols/wlm/wlmchatsession.cpp	2014-05-29 02:23:08.743392380 +1100
@@ -557,7 +557,11 @@
                 screenColourmap);
 
     if (status != GIF_OK) {
+#if GIFLIB_MAJOR * 100 + GIFLIB_MINOR >= 501
+        EGifCloseFile(GifFile, NULL);
+#else
         EGifCloseFile(GifFile);
+#endif
         return;
     }
 
@@ -580,11 +584,19 @@
 
     if (status != GIF_OK) {
         printGifErrorMessage();
+#if GIFLIB_MAJOR * 100 + GIFLIB_MINOR >= 501
+        EGifCloseFile(GifFile, NULL);
+#else
         EGifCloseFile(GifFile);
+#endif
         return;
     }
 
+#if GIFLIB_MAJOR * 100 + GIFLIB_MINOR >= 501
+    if (EGifCloseFile(GifFile, NULL) != GIF_OK) {
+#else
     if (EGifCloseFile(GifFile) != GIF_OK) {
+#endif
         printGifErrorMessage();
         return;
     }
