--- kopete/protocols/oscar/liboscar/tasks/chatnavservicetask.cpp.omv~	2017-11-03 18:47:41.712115044 +0100
+++ kopete/protocols/oscar/liboscar/tasks/chatnavservicetask.cpp	2017-11-03 18:47:52.381092851 +0100
@@ -103,7 +103,7 @@ bool ChatNavServiceTask::take( Transfer*
 void ChatNavServiceTask::onGo()
 {
     FLAP f =  { 0x02, 0, 0x00 };
-    SNAC s = { 0x000D, m_type, 0x0000, client()->snacSequence() };
+    SNAC s = { 0x000D, static_cast<WORD>(m_type), 0x0000, client()->snacSequence() };
     Buffer* b = new Buffer();
 
     Transfer* t = createTransfer( f, s, b );
